src/Kunstmaan/AdminListBundle/AdminList/FilterType/Traits/BooleanFilterTrait.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Kunstmaan\AdminListBundle\AdminList\FilterType\Traits;
+
+use Symfony\Component\HttpFoundation\Request;
+
+trait BooleanFilterTrait
+{
+    /**
+     * @param Request $request  The request
+     * @param array   &$data    The data
+     * @param string  $uniqueId The unique identifier
+     */
+    public function bindRequest(Request $request, array &$data, $uniqueId)
+    {
+        $data['value'] = $request->query->get('filter_value_' . $uniqueId);
+    }
+
+    /**
+     * @param array  $data     The data
+     * @param string $uniqueId The unique identifier
+     */
+    public function apply(array $data, $uniqueId)
+    {
+        if (isset($data['value'])) {
+            switch ($data['value']) {
+                case 'true':
+                    $this->queryBuilder->andWhere($this->queryBuilder->expr()->eq($this->getAlias() . $this->columnName, 'true'));
+                    break;
+                case 'false':
+                    $this->queryBuilder->andWhere($this->queryBuilder->expr()->eq($this->getAlias() . $this->columnName, 'false'));
+                    break;
+            }
+        }
+    }
+
+    /**
+     * @return string
+     */
+    public function getTemplate()
+    {
+        return 'KunstmaanAdminListBundle:FilterType:booleanFilter.html.twig';
+    }
+}
```

Comment:
Can you add newlines to the end of your files, there are several once with this issue. :)